tutorials/rag/evaluate-rag-flow.ipynb

```diff
@@ -248,7 +238,7 @@
     "\n",
     "def call_llm(**inputs):\n",
     "    # Populate the Prompt template\n",
-    "    messages = populate_template(template, inputs)\n",
+    "    messages = humanloop.prompts.populate_template(template, inputs)\n",
```

Comment:
i intentionally didn't do this as defining a reference RAG pipeline already using humanloop feels like it's slightly cheating.

but am fine with this change; is minor and does make it cleaner.

Reply:
fair